runtime/hydradx/src/governance.rs

```diff
@@ -284,6 +292,7 @@ impl pallet_elections_phragmen::Config for Runtime {
 	type MaxCandidates = MaxElectionCandidates;
 	type MaxVoters = MaxElectionVoters;
 	type WeightInfo = ();
+	type MaxVotesPerVoter = MaxVotesPerVoter;
```

Comment:
doesn't this break current votes. for ppl that voted for more than 5 candidates?

Reply:
there are already multiple accounts that voted for more than 5 candidates